buzzard.py

```diff
@@ -1106,8 +1116,8 @@ def simplify(points, tolerance, highestQuality):
     parser.add_argument('-v', dest='verbose', default=False,
                         help='Verbose mode (helpful for debugging)', action='store_true')
 
-    parser.add_argument('-o', dest='outMode', default='b', choices=['b', 'ls', 'lib', 'ki'],
-                        help='Output Mode (\'b\'=board script, \'ls\'=library script, \'lib\'=library file, \'ki\'=KiCad footprint)')
+    parser.add_argument('-o', dest='outMode', default='b', choices=['b', 'ls', 'lib', 'ki', 'ki5'],
+                        help='Output Mode (\'b\'=board script, \'ls\'=library script, \'lib\'=library file, \'ki\'=KiCad v6 footprint, \'ki5\'=Kicad v5 footprint)')
```

Comment:
Very small nit-pick here.

Just on the capitalization of KiCad, to match the v6 description
```diff
- \'ki5\'=Kicad v5 footprint`
+ \'ki5\'=KiCad v5 footprint`
```

Reply:
@gregdavill Done !